src/rubrix/__init__.py

```diff
@@ -69,6 +70,8 @@ def init(
             it will default to `http://localhost:6900`.
         api_key: Authentification key for the REST API. If `None` (default) and the env variable ``RUBRIX_API_KEY``
             is not set, it will default to `rubrix.apikey`.
+        workspace: Active user workspace used for client session. If `None` (default) and the
+            env variable ``RUBRIX_WORKSPACE`` is not set, it will default to user workspace
```

Comment:
```suggestion
        workspace: The workspace to which records will be logged/loaded. If `None` (default) and the 
            env variable ``RUBRIX_WORKSPACE`` is not set, it will default to the private user workspace.
```